ci/delete_thin-edge/delete_thin_edge.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -euo pipefail
+
+usage() {
+    cat <<EOF
+USAGE:
+    delete_thin_edge [remove/purge]
+EOF
+}
+
+stop_a_service_if_running() {
+    local status
+    status=$(systemctl show -p ActiveState --value "$1")
+    if [ "$status" == "active" ]; then
+        sudo systemctl stop "$1"
+    fi
+}
+
+stop_services() {
+    stop_a_service_if_running "tedge-watchdog.service"
+    stop_a_service_if_running "tedge-mapper-collectd.service"
+    stop_a_service_if_running "c8y_log_plugin"
+    stop_a_service_if_running "c8y_configuration_plugin"
+    stop_a_service_if_running "apama"
+}
+
+remove_or_purge_package_if_exists() {
+    local status
+    status=$(dpkg -s "$2" | grep -w installed) && returncode=$? || returncode=$?
+    if [ "$status" == "Status: install ok installed" ]; then
+        sudo apt --assume-yes "$1" "$2"
+    fi
+}
+
+remove_packages() {
+    remove_or_purge_package_if_exists "remove" "tedge"
```

Comment:
I think here one line comment will be appreciated why `tedge_mapper`, `tedge_agent`, and `tedge_watchdog` don't need to be listed. (Explain the dependency shortly after this line)

This is the dependencies around here.

1. tedge
2. tedge_mapper (dependency on `tedge`)
3. tedge_agent (dependency on `tedge_mapper`)
4. tedge_watchdog (dependency on `tedge`)

Others plugin don't have any dependencies, therefore we need to run the command explicitly.

Reply:
Added a comment.